src/ev3_bluetooth.c

```diff
@@ -0,0 +1,332 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <poll.h>
+#include <ctype.h>
+#include "../firmware_headers/bluetooth/bluetooth.h"
+#include "../firmware_headers/bluetooth/hci.h"
+#include "../firmware_headers/bluetooth/hci_lib.h"
+#include "../firmware_headers/bluetooth/rfcomm.h"
+
+#include "ev3_bluetooth.h"
+
+/**
+ * Channel 1 is already taken by the UI process of the robot, so we use 2.
+ */
+#define DEFAULT_C4EV3_RFCOMM_CHANNEL    2
+#define MAX_CONNECTIONS                 7
+#define SHORT_BLUETOOTH_ADDRESS_LENGTH  sizeof("000000000000")
+
+#define KNOWN_BLUETOOTH_NAMES_FILE_NAME_LENGTH 256
+#define KNOWN_BLUETOOTH_NAMES_FILE_ROW_LENGTH  256
+
+void BluetoothInit () { }
```

Comment:
I would leave it out if it's empty

Reply:
Nevermind, I didn't see that it's exported in the header. Leave it in, but use a prototype.

Reply:
I'm sorry but I don't understand what "use a prototype" mean, can you make an example?
The function is defined in the header file, and I created it to maintain the consistency between the other modules (which all have Init/Exit functions). However it doesn't do anything, so I think we can remove it.

Reply:
In C (not C++) an empty set of parenthesis means an unspecified number of parameters. So a user can call `BluetoothInit()`, `BluetoothInit(1)` or `BluetoothInit("%s\b", "hello world")` without an error.

You can disallow this by using a prototype, i.e. `void BluetoothInit(void)`

Reply:
Thank you, I didn't understand. I specified (void) for all functions without parameters.